services/docker-compose.yml

```diff
@@ -130,7 +130,7 @@ services:
     env_file:
       - services/net/image/.env
     ports:
-      - ${CONTENT_PORT:-40027}:8081
+      - ${IMAGE_PORT:-40027}:8081
```

Comment:
I believe Curtis resolved this already, you may need to rebase.

Reply:
I fixed it before Curtis :-)